_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -33,22 +33,22 @@ acknowledgments:
 
 This rule applies to any element with an `aria-hidden` [attribute value][] of `true`.
 
-**Note:** Using `aria-hidden="false"` on a descendant of an element with `aria-hidden="true"` **does not** expose that element. `aria-hidden="true"` hides itself and all its content from assistive technologies.
-
 ## Expectation
 
-None of the target elements are part of [sequential focus navigation](https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation), nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are part of [sequential focus navigation](https://html.spec.whatwg.org/#sequential-focus-navigation).
+None of the target elements can be [focused][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that can be [focused][].
```

Comment:
How about we do something like this:

```suggestion
None of the target elements are [focusable][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [focusable][].
```

```md
---
title: Focusable
key: focusable
unambiguous: true
objective: true
input_aspects:
  - CSS styling
  - DOM Tree
---

An element is _focusable_ if one or both of the following are true:

- the element is part of [sequential focus navigation][]; or
- the element can be [focused][] with a [single pointer][] user action in the [visible][] region of the [page][].

**Exception**: Elements that lose focus within 100ms of receiving focus are not considered _focusable_.

**Note**: The 100ms time span is an arbitrary limit which is not included in WCAG. Any focus redirect that is slower than 100ms will be clearly noticeable and could be considered a [change of context][] instead.

[change of context]: https://www.w3.org/TR/WCAG21/#dfn-change-of-context
[focused]: #focused
[page]: #html-web-page
[single pointer]: https://www.w3.org/TR/WCAG21/#dfn-single-pointer
[sequential focus navigation]: https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation
[visible]: #visible
```

Reply:
@WilcoFiers When processing this I recalled that we already have a [Focusable definition](https://act-rules.github.io/glossary/#focusable)

Still, checking the HTML spec, your definition looks a lot like the two types of focusability: 
- [Sequentially focusable](https://html.spec.whatwg.org/#sequentially-focusable)
- [Click focusable](https://html.spec.whatwg.org/#click-focusable)

Would you agree with rewriting the expectation as 
```suggestion
None of the target elements are [sequentially focusable](https://html.spec.whatwg.org/#sequentially-focusable) or [click focusable](https://html.spec.whatwg.org/#click-focusable), nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [sequentially focusable][] or [click focusable](https://html.spec.whatwg.org/#click-focusable).
```

Reply:
> A focusable area is said to be click focusable if the user agent determines that it is click focusable

No... that definitely doesn't work for me. Besides, this just again says that anything with negative tabindex is click focusable, which isn't what we want. If you can't actually click it, because it's off screen or obscured, this shouldn't be true.

Reply:
@WilcoFiers 
Then before committing to changing the definition, just want to make sure you (and everyone else) agree with this course of action:
- update the name of the existing ["Focusable" definition](https://act-rules.github.io/glossary/#focusable) to "Keyboard focusable" and update the rules that used this definition (this should be done in a different PR)
- create the new "Focusable" definition (in this PR)
- update any of the rules that used the old focusable definition to use the new one if that makes sense (rules that looked specifically at keyboard focus won't be updated) (this is another PR after this one has been merged)